src/mongo_logger.ts

```diff
@@ -738,6 +750,37 @@ export class MongoLogger {
     return compareSeverity(severity, this.componentSeverities[component]) <= 0;
   }
 
+  turnOffSeverities() {
+    for (const key of Object.keys(MongoLoggableComponent)) {
+      this.componentSeverities[key as MongoLoggableComponent] = SeverityLevel.OFF;
+    }
+  }
+
+  private logWriteFailureHandler(error: Error) {
+    try {
+      if ((this.logDestination as any)?.streamName === 'stderr') {
```

Comment:
I think this points to a miss in the type of logDestination. I don't think we ever have Writable, it is always a MongoDBLogWritable, but I also asked in another comment that we track this stderr information on this logger class, not the destination.

Reply:
I was thinking the same actually. I'll change the internal logger structure to just have `MongoDbLogWritable` as the type.